src/lib/radio/radio.spec.ts

```diff
@@ -7,7 +7,7 @@ import {dispatchFakeEvent, FakeViewportRuler} from '@angular/cdk/testing';
 import {RIPPLE_FADE_IN_DURATION, RIPPLE_FADE_OUT_DURATION} from '@angular/material/core';
 import {MdRadioButton, MdRadioChange, MdRadioGroup, MdRadioModule} from './index';
 
-describe('MdRadio', () => {
+fdescribe('MdRadio', () => {
```

Comment:
The `describe` has been added accidentally here.

Reply:
Ops, good catch.